src/clr-angular/popover/dropdown/dropdown-trigger.ts

```diff
@@ -17,16 +18,24 @@ import { ClrDropdown } from './dropdown';
     '[class.dropdown-item]': '!isRootLevelToggle',
     '[class.expandable]': '!isRootLevelToggle',
     '[class.active]': 'active',
+    '[attr.aria-haspopup]': '"menu"',
+    '[attr.aria-expanded]': 'ifOpenService.open',
```

Comment:
We need to apply the same fix as #3450.

Reply:
Absolutely. I'm also merging fixes from https://github.com/vmware/clarity/pull/3440 into this PR.